static/styles/left_sidebar.css

```diff
@@ -175,7 +175,6 @@ li.show-more-topics {
 #add-stream-link {
     text-decoration: none;
     margin-left: 10px;
-    margin-bottom: 18px;
```

Comment:
This was intentional (#12519), although I’m not sure that reasoning (still) applies, since browsers move the URL display from the lower-left corner to the lower-right corner when the pointer gets close.

Reply:
Yes, I have tested this and when the cursor approaches the url, the url shifts to the right side.

Reply:
I don't see that exact behavior in my version of Chrome on Linux -- instead, the URL shifts downwards in a way that makes it eventually go mostly or entirely offscreen.  In any case, it does appear that most browsers have resolved this issue somewhat, and in any case, I think having this be fixed in an area below the scrollable region is an alternative solution to the problem from #12519.

The other thing I noticed while playing with this PR is the button's vertical position currently behaves poorly with browser zoom in -- if you zoom in a couple ticks (as I did to force a scrollbar with a relatively large window size and the default development environment streams), the button ends up with a weirdly large gap below it compared to that below it.  I think we probably want it to be vertically centered within the space allocated to it, or something.